docs/get-started.md

```diff
@@ -209,6 +209,40 @@ Commands that follow this style are:
 
 For machines other than `default`, and commands other than those listed above, you must always specify the name explicitly as an argument.
 
+## Start local machines on startup
+
+Some users like to embed `eval $(docker-machine env default)` on their shell profiles, so docker client gets automatically configured at the start of each shell session. This can print errors if the `default` machine hasn't been started yet. You can configure your system to start the default machine automatically during the startup.
```

Comment:
Please revise to:

> In order to ensure that the Docker client is automatically configured at the start of each shell session, some users like to embed `eval $(docker-machine env default)` in their shell profiles (e.g. the `~/.bash_profile` file). However, this will fail if the `default` machine is not running.  If desired, you can configure your system to start the `default` machine automatically.